test/README.md

```diff
@@ -0,0 +1,62 @@
+# Hedera Smart Contracts - tests
+Test scenarios verifying the correct behavior of the following contracts:
+
+- Diamond
+- ERC-20
+- ERC-721
+- ERC-1155
+- HIP-583
+- HTS-Precompile [ERC-20, ERX-721, token-create, token-management, token-query, token-transfer]
+- SafeHTS
+
+# Hedera Smart Contracts - Local test suite execution
+The tests are implmented in an independent manner, so they can be executed all, by test set or a single test.
+
+## Using command line:
+Running "npx hardhat test" in the terminal will execute all the existing tests. If you want to execute specific test set or single test you should:
+1. Open the desired test suite - Ex. ERC20.js
+2. Add "only" to the desired describe/it - Ex. describe.only || it.only
+3. Run in the terminal "npx hardhat test"
```

Comment:
Q: is there not a way to filter tests by file through command line?
That would be better if possible than devs having to change code